pkg/k8s/watchers/pod.go

```diff
@@ -731,6 +742,37 @@ func (k *K8sWatcher) deleteHostPortMapping(pod *slim_corev1.Pod, podIPs []string
 	return nil
 }
 
+func findServiceByL3n4Addr(l3n4Addr loadbalancer.L3n4Addr, svcs []*loadbalancer.SVC) *loadbalancer.SVC {
+	for _, svc := range svcs {
+		if l3n4Addr.DeepEqual(&svc.Frontend.L3n4Addr) {
+			return svc
+		}
+	}
+	return nil
+}
+
+func deepEqualBackends(backends1 []*loadbalancer.Backend, backends2 []*loadbalancer.Backend) bool {
+	if len(backends1) != len(backends2) {
+		return false
+	}
+
+	l3n4AddrMap := make(map[string]*loadbalancer.Backend)
```

Comment:
The slice is known, so might as well use it?

Reply:
Not sure which slice you're referring to?